### DISCUSSION TITLE
composer.lock в демо проекте

### DISCUSSION THREAD

**Discussion by shevl-dev** [moonshine-software/moonshine#1097] [Ideas]
Так как в **moonshine-software/demo-project** заигнорен файл **composer.lock**, а версия moonshine в нём развернута не самая свежая, то при установке зависимостей получается вот это, т.к. ассеты в демо проекте не соответствуют ассетам самой последней версии moonshine:

![image](https://github.com/moonshine-software/moonshine/assets/118539450/c0852635-b121-4189-8572-b3cd7e4bd778)

Собственно предлагаю убрать composer.lock из .gitignore

**Comment by lee-to** [comment]
Да разные проблемы встречаются, в целом можно сразу делать pr, любая адекватная идея будет одобрена

**Reply by shevl-dev** [reply]
Меня только смущает, не будет ли конфликта у тех, кто уже развернул проект и решит выполнить composer update, у него уже есть сгенерированный файл composer.lock, а с гита придёт другой.
Ну и нужно ведь обновить мун, правильно сгенерировать ассеты и т.п., я не стал пока в это лезть )

**Reply by lee-to** [reply]
composer update пофигу на лок
